### PR TITLE
Report draft

### PR DESCRIPTION
**Pull request by katyhuff**
Hi Folks, please take a look at this pdf to review this PR. Plenty is missing, but hopefully nothing is incorrect. 

[2021-04-nm-illinois.pdf](https://github.com/arfc/2021-04-nm-illinois/files/6406185/2021-04-nm-illinois.pdf)

I'll probably merge this in a few hours, unless there are objections. Before then, please feel free to make comments here with suggestions or make PRs thereafter correcting anything you see fit. Accepting changes through monday.

### CODE REVIEW

**Comment by samgdotson** [comment]
I know this is still a working document but it's coming along nicely. I agree with all of the conclusions and summary points. My biggest comments were about the description of the objective function, which I think should be a double sum. Maybe that's too picky.

[DRAFT-2021-04-nm-illinois_dotson_comments.pdf](https://github.com/arfc/2021-04-nm-illinois/files/6406912/DRAFT-2021-04-nm-illinois.pdf)

File: report/discussion.tex [hunk 1]
```diff
@@ -1,2 +1,43 @@
...
+system spanning the 2020-2050 timeframe. The linear programming model 
+identified energy mixtures which minimized total system costs 
+in the context of those potential technology, economic, and policy constraints.
+
+This work accordingly adds to the growing body of research demonstrating how 
+decommissioning existing, emissions-free nuclear power plants endangers the 
+feasibility of near-term zero-emissions targets.  Our conclusions are 
+consisistent with and confirmatory of such literature, in particular, the 
+February 2021 National Academy of Sciences, Engineering, and Medicine consensus 
+report, ``Accelerating Decarbonization of the U.S. Energy System,'' which determined unequivocally that US decarbonization will require keeping existing nuclear plants open
```
Comment (npanczyk) [inline report/discussion.tex:14]:
```suggestion
report, ``Accelerating Decarbonization of the U.S. Energy System,'' which determined unequivocally that U.S. decarbonization will require keeping existing nuclear plants open
```

File: report/discussion.tex [hunk 2]
```diff
@@ -1,2 +1,43 @@
...
+\cite{national_academies_of_sciences_engineering_and_medicine_2021_accelerating_2021}.
+
+Specifically, our simulations indicate that decarbonization
+in Illinois will require not only maintenance but expansion of nuclear energy capacity.
+When the 2020-2050 cost of the Illinois electric system is minimized, 
+comparison of these twelve (12) scenarios showed that :
```
Comment (npanczyk) [inline report/discussion.tex:20]:
```suggestion
comparison of these twelve (12) scenarios showed that:
```

File: report/summary.tex [hunk 3]
```diff
@@ -1,101 +1,54 @@
...
-2021 National Academy of Sciences, Engineering, and Medicine report, 
-``Accelerating Decarbonization of the U.S. Energy System,'' which determined 
-unequivocally that US decarbonization will require keeping existing nuclear 
-plants open \cite{national_accelerating_2021}.
+Previous energy systems research has shown that such clean energy goals cannot 
+be reached if nuclear plants prematurely retire 
+\cite{national_academies_of_sciences_engineering_and_medicine_2021_accelerating_2021,larson_net-zero_2020,davis_net-zero_2018}.  
+In particular, the February 2021 National Academy of Sciences, Engineering, and 
+Medicine consensus report, ``Accelerating Decarbonization of the U.S. Energy 
+System,'' determined unequivocally that US decarbonization will require keeping
```
Comment (npanczyk) [inline report/summary.tex:14]:
```suggestion
System,'' determined unequivocally that U.S. decarbonization will require keeping 
```

**Review by npanczyk — APPROVED** [review]
This looks great to me! Made mostly little grammatical notes. For the sake of expediency, I didn't change as much passive voice as I could have. That being said, happy to go back through if that's a major concern. I personally think it's fine as is. Only other comment I have that I couldn't note in the files is that Tables 2 and 3 have a comment below their titles: "Scenario with the lowest... is in bold." and the formatting is a little funky. Just wondering if it would be possible to add a space between that and the bar or maybe make the font smaller so the tables don't look so cramped. 

That's it from me, though! Let me know if there's anything else you need me for!

**Review by munkm — APPROVED** [review]
This is looking great! I have a few minor suggestions and comments that I think will help the readers of this report. 

I noticed also that the figure of Lifecycle Carbon Emissions (Figure 8 in the PDF) says that it is plotting all scenarios, but doesn't include the XNn scenarios in the legend. Either the figure label or the figure should be updated to either remedy the omission or clarify why it isn't included in the figure.

**Comment by abachma2** [comment]
Looks got to me so far. I made a few comments on the pdf, so they may be older than some of the recent commits. 
[2021-04-nm-illinois-Bachmann.pdf](https://github.com/arfc/2021-04-nm-illinois/files/6415587/2021-04-nm-illinois-Bachmann.pdf)

**Review by abachma2 — APPROVED** [review]

File: report/methods.tex [hunk 4]
```diff
@@ -16,38 +18,67 @@ \subsection{Optimization Analysis}
...
 the Intergovernmental Panel on Climate Change 
-\cite{intergovernmental_panel_on_climate_change_annex_2014,intergovernmental_panel_on_climate_change_climate_2014,intergovernmental_panel_on_climate_change_climate_2014-1,intergovernmental_panel_on_climate_change_climate_2014-2}
+\cite{intergovernmental_panel_on_climate_change_annex_2014,intergovernmental_panel_on_climate_change_climate_2014,intergovernmental_panel_on_climate_change_climate_2014-1,intergovernmental_panel_on_climate_change_climate_2014-2},
 the Interstate Renewable Energy Council 
-\cite{sherwood_us_2009,sherwood_us_2010,sherwood_us_2011,brown_solid_1996,sherwood_us_2012,sherwood_us_2013,sherwood_us_2014}
-and the Department of Energy's EERE and NE offices 
-\cite{us_department_of_energy_capital_2016}, the National Renewable Energy 
-Laboratory \cite{nrel_national_renewable_energy_laboratory_2020_2020}.
+\cite{sherwood_us_2009,sherwood_us_2010,sherwood_us_2011,brown_solid_1996,sherwood_us_2012,sherwood_us_2013,sherwood_us_2014}, 
+the Department of Energy's EERE and NE offices
```
[the remark this answers is not in the document]
Comment (katyhuff) [reply report/methods.tex:78]: this is handled in later commit.

**Pull request merged**